### ISSUE TITLE
Support colorblindness

### ISSUE THREAD

**Issue by natewatson999** [UKHomeOffice/posters#53]
Right now, users with low vision and screenreaders are covered. I think a version of the poster should be made to help accommodate users with various kinds of colorblindness. Some of these are covered elsewhere. I've listed some suggestions:
Do:
* Use black and white for text-color/background-color combinations.
* Provide a high contrast mode for when the commonly preferred color palate is hard to read for some.
* Use aliased fonts. 
* Use geometrically simple lines. This is for rarer forms of colorblindness which are not covered here. The idea is the following: While the users may still struggle to perceive the content with the chosen palate, simple lines are easier to to trace in low-contrast situations. 
* Use supplemental labels. Don't just have the hot wire red and neutral wire blue. Label the hot red wire as "hot" and label the blue neutral wire "neutral". This is for people who only see in greyscale. 
Dont:
* Use exclusive color coding as a form of labeling. If a person can only see in greyscale, and the hot and neutral wires are only differentiated by color, they could die. 
* Use grey text on a white background, or the reverse. 
* Use red and green, green and brown, or brown and red; as combinations of text color and background color.
* Use dark colors on black. Anything with a combined color rating of 255 or less between red, green, and blue should be considered dark for this rule.
* Use cyan and magenta as text-color/background-color combinations.
* Use green and yellow as text-color/background-color combinations.
* Use orange and yellow as text-color/background-color combinations.
* Use orange and red as text-color/background-color combinations.
* Use purple and blue as text-color/background-color combinations. If the purple is at least 2 parts red, this rule can be ignored, but only if the blue is not Cyan.

**Comment by karypun** [comment]
@natewatson999 these are great! Let me have a discussion with my team about this and we'll keep in touch.

**Comment by KeithRhodes** [comment]
@natewatson999 @karypun 
I have red-green colour deficiency, and I found all the information on the low-vision poster to be appropriate.

I especially agree that the use of colour alone to convey meaning should be discouraged. My rule is that meaning can be conveyed by colour, position, shape, and should use at least two of those three elements.

The term "hot" for a conductor is not British English. "Hot" is a US term; in the UK it is the "phase" (technical term) or "live" (common term).

**Comment by natewatson999** [comment]
@KeithRhodes , yes, I used the american wording for electrical wiring. That should probably be reworded for the British dialect family. Other examples might be a good idea as well: Purely color-coding medication bottles would theoretically kill somebody, hence why text labels are also used. "adjust spelling as appropriate".

**Comment by karypun** [comment]
@KeithRhodes @natewatson999 I'm on annual leave at the moment so apologies for not getting back sooner. However, your points about labelling colours, in this case, wires as "hot" versus "live"/"phase" is interesting as it involves regional differences in language which can confuse some people.  

Perhaps labelling could be redefined to be more universal. Instead of describing wires in their respective states, we could describe them as "dangerous" or "safe". Just an idea.. but this is great to be able to discuss this.

**Comment by Ryuno-Ki** [comment]
IIRC you should use a high enough contrast with respect to the _hue_ of a colour (as denotated in HSL).

Blue-yellow can have an even better contrast than black-on-white. I can look up my sources if needed.

**Comment by achecht** [comment]
Suggested addition: "Never use color as the sole means of communicating information."
Example: A chart or table in which you refer to the "Green" items/graph as being [this] and the "Red" items/graph as being [that].

**Comment by karypun** [comment]
Thanks for opening this thread and suggesting colour blindness as another poster.  We decided to stop creating or adding new posters to the accessibility set but are looking to provide a template for others to use so they can create posters of their own. We'll let you know once that's done. In the meantime, we'll continue to accept translations of the existing posters for now.

**Comment by achecht** [comment]
Thanks. I noticed the English posters were using UK spellings. Would you be open to creating versions with American spellings of I sent those changes?

~Alan Hecht
(sent from my phone)

On Jun 4, 2018, at 3:26 PM, Karwai Pun <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks for opening this thread and suggesting colour blindness as another poster. We decided to stop creating or adding new posters to the accessibility set but are looking to provide a template for others to use so they can create posters of their own. We'll let you know once that's done. In the meantime, we'll continue to accept translations of the existing posters for now.

**Comment by karypun** [comment]
Hi, we're not creating any further posters but feel free to use our template in sketch. We've been seeing lots of wonderful posters and we'd love to see what others come up with. https://github.com/UKHomeOffice/posters/tree/master/accessibility/dos-donts/posters_en-UK
